Gopkg.toml

```diff
@@ -53,3 +57,46 @@
 [[constraint]]
   name = "github.com/docker/go"
   version = "1.5.1-1"
+
+
+[[constraint]]
+  name = "github.com/docker/cnab-to-oci"
+  branch = "master"
+
+[[constraint]]
+  name = "github.com/deislabs/cnab-go"
+  branch = "master"
+
+[[override]]
+  name = "github.com/containerd/containerd"
+  source = "github.com/simonferquel/containerd"
+  revision = "42c3614d58767e96c1752977fed94d0b140a0173"
+
+
+[[override]]
+  name = "golang.org/x/sys"
+  revision = "f49334f85ddcf0f08d7fb6dd7363e9e6d6b777eb"
+
+[[override]]
+  name = "k8s.io/apimachinery"
```

Comment:
Something needs to be cleaned in docker/cli, you shouldn't have to vendor kubernetes just because you want to push and pull...

Reply:
Yeah, I ended up having to pinpoint this because of `docker/cli`...

Reply:
I'll try to dig and find the culprit in docker/cli 👍

Reply:
That would be fantastic!
Let me know how that works, thanks!

Reply:
@silvin-lubecki  - any updates on solving this in `docker/cli`?
Thanks!

Reply:
@radu-matei 2 PR's were merged on docker/cli to remove some dependencies while vendoring the code:
https://github.com/docker/cli/pull/1887 and https://github.com/docker/cli/pull/1890 .
Can you bump docker/cli to master on this PR and check you have less dependencies? 🙏 
By the way docker/cli 19.03 will be release soon, I think we will should pin this version as soon as it is available.

Reply:
Thanks for the update, will bump th  CLI dependency.